cli/http_cache.rs

```diff
@@ -81,6 +82,8 @@ pub struct HttpCache {
 pub struct Metadata {
   pub headers: HeadersMap,
   pub url: String,
+  #[serde(default = "SystemTime::now")]
```

Comment:
Had to do this because we are changing the format of the metadata and any existing cached metadata would be broken.  This has minimal impact on figuring out if the cached value is valid or not.

Reply:
Just out of curiosity, can we not use the `Date` header for this?

Reply:
The reference uses the timestamp when the response was received, versus whatever the response attests to in it date header.  I didn't dig into what is more correct, it does seem logical that when calculating time in cache that you would use "your" time versus the server's time.

Reply:
Well the `age` / `expiration` are relative to the server time, as they calculated them, right?

Reply:
age is the age header plus resident time, resident time is calculated "local" time.  Expiration is calculated off raw server time (which uses the "date" header and the "local" time as a backup).